esp/src/src-react/components/XrefDirectories.tsx

```diff
@@ -27,15 +27,15 @@ export const XrefDirectories: React.FunctionComponent<XrefDirectoriesProps> = ({
         sort: { attribute: "name", descending: false },
         filename: "xrefsDirectories",
         columns: {
-            name: { width: 100, label: nlsHPCC.Name },
-            num: { width: 30, label: nlsHPCC.Files },
-            size: { width: 30, label: nlsHPCC.TotalSize },
-            maxIP: { width: 30, label: nlsHPCC.MaxNode },
-            maxSize: { width: 30, label: nlsHPCC.MaxSize },
-            minIP: { width: 30, label: nlsHPCC.MinNode },
-            minSize: { width: 30, label: nlsHPCC.MinSize },
+            name: { width: 300, label: nlsHPCC.Name },
+            num: { width: 300, label: nlsHPCC.Files },
```

Comment:
should this one also be 100, like the other more narrow columns?

Reply:
@jeclrsg no, when testing those two columns required more space than the others by default.

Reply:
The "num" / "Files" column was the one I was talking about.

![image](https://github.com/hpcc-systems/HPCC-Platform/assets/79224539/74815f74-d71a-4d7f-b9ab-d15ab07677a2)

Reply:
I updated the Files to 100 and expanded the Name field so users are able to see full  path name.